rllib/core/learner/tests/test_learner_group.py

```diff
@@ -71,13 +82,22 @@ def local_training_helper(self, fw, scaling_mode) -> None:
 
         # make the state of the learner and the local learner_group identical
         local_learner.set_state(learner_group.get_state())
+        # learner_group.set_state(learner_group.get_state())
+        check(local_learner.get_state(), learner_group.get_state())
 
         # do another update
         batch = reader.next()
         ma_batch = MultiAgentBatch(
             {new_module_id: batch, DEFAULT_POLICY_ID: batch}, env_steps=batch.count
         )
-        check(local_learner.update(ma_batch), learner_group.update(ma_batch))
+        local_learner_results = local_learner.update(ma_batch)
+        learner_group_results = learner_group.update(ma_batch)
+
+        check(local_learner.get_state(), learner_group.get_state())
+        local_learner_results = local_learner.update(ma_batch)
+        learner_group_results = learner_group.update(ma_batch)
```

Comment:
Could we leave these out and just do:

```
local_learner_results = local_learner.update(ma_batch)
learner_group_results = learner_group.update(ma_batch)

check(local_learner.get_state(), learner_group.get_state())
check(local_learner_results, learner_group_results)
```
?

Reply:
I've intentionally done this because the optimizer state is not initialized fully until the first time that training is completed.

Reply:
Cool! That's not clear from reading the test code. So my nit would be to add a comment here that explains the test a little.

Reply:
done